cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -4157,4 +4157,11 @@
       Internal directory that maintains the currently disabled and enabled configurations
     </description>
   </property>
+  <property>
+    <name>capability.autoinstall.threads</name>
+    <value>5</value>
+    <description>
+      Number of threads to be used for auto installing resources when a capability is enabled
```

Comment:
Is the minimum 1 or 0? Maybe we should specify if 0 is a valid option (meaning, autoinstall is disabled in this case).

Reply:
@fernst I don't think we need to explicitly disable autoinstall here because it's not enabled by default. Autoinstall is enabled only if a capability configuration is present. Also, the capability framework does support capability deletion; autoinstall will be disabled if the capability configuration is removed.